models/cdm/examples/synthetic/cdm.py

```diff
@@ -169,11 +168,16 @@ def makeStations(self):
         Create a set of station coordinate
         """
         # get some help
-        import itertools
-        # build a set of points on a grid
-        stations = itertools.product(range(-5,6), range(-5,6))
+        import numpy as np
```

Comment:
I think for the synthetic case, this is better to constrain the number of points.